PlannedToAT/appsettings.json

```diff
@@ -5,23 +5,5 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "ConnectionStrings": {
-    "DefaultConnection": "hostname=plannedtoatscus.mysql.database.azure.com,port=3306,username=Jamy,password=Faye2011,;"
-  },
-  "AllowedHosts": "*",
-  "sqltools.connections": [
-    {
-        "mysqlOptions": {
-            "authProtocol": "default",
-            "enableSsl": "Disabled"
-        },
-        "previewLimit": 50,
-        "server": "localhost",
-        "port": 3306,
-        "driver": "MySQL",
-        "name": "plannedtoat",
-        "username": "Jamy",
-        "database": "plannedtoat"
-    }
-]
-}
```

Comment:
should never have secrets in the appsettings.  + these were overridding what was in the appsettings.Development.json so nothing was running locally.  This will need to be defined as a secret in Azure and as a environment variable for the host